lib/rucio/daemons/reaper/reaper.py

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
the empty line here is actually a problem since the encoding below needs to be the first line

Reply:
Done.